### PR TITLE
Additional Blob/Datalake regression storage pipeline fix

### PR DESCRIPTION
**Pull request by tasherif-msft**
There was another regression failure related to the migration to the new generator. This PR should resolve it.

### CODE REVIEW

File: eng/pipelines/templates/jobs/archetype-sdk-client.yml [hunk 1]
```diff
@@ -178,7 +178,6 @@ jobs:
               BuildTargetingString: ${{ parameters.BuildTargetingString }}
 
   - job: 'RunRegression'
-    condition: and(succeededOrFailed(), or(eq(variables['Run.Regression'], 'true'), and(eq(variables['Build.Reason'], 'Schedule'), eq(variables['System.TeamProject'],'internal'))))
```
Comment (weshaggard) [inline eng/pipelines/templates/jobs/archetype-sdk-client.yml:181]: @scbedd @praveenkuttappan I assume we don't want to remove this condition.

Comment (tasherif-msft) [reply eng/pipelines/templates/jobs/archetype-sdk-client.yml:181]: I'm removing it temporarily - for testing purposes (forcing regression to run). I will undo this before merging.

File: eng/tox/install_depend_packages.py [hunk 2]
```diff
@@ -28,7 +28,8 @@
 logging.getLogger().setLevel(logging.INFO)
 
 MINIMUM_VERSION_SUPPORTED_OVERRIDE = {
-    'azure-common': '1.1.10'
+    'azure-common': '1.1.10',
+    'azure-storage-file-datalake': ' 12.2.0'
```
Comment (scbedd) [inline eng/tox/install_depend_packages.py:32]:
```suggestion
    'azure-storage-file-datalake': '12.2.0'
```

**Review by scbedd — COMMENTED** [review]
I'll approve after condition gets add back!

File: eng/tox/install_depend_packages.py (hunk 2 above)
Comment (tasherif-msft) [reply eng/tox/install_depend_packages.py:32]: woops

**Review by scbedd — APPROVED** [review]

**Pull request merged**